opentelemetry-otlp/src/logs.rs

```diff
@@ -103,6 +103,10 @@ impl opentelemetry_sdk::export::logs::LogExporter for LogExporter {
     async fn export(&mut self, batch: Vec<LogData>) -> opentelemetry::logs::LogResult<()> {
         self.client.export(batch).await
     }
+
+    fn set_resource(&mut self, _resource: &opentelemetry_sdk::Resource) {
+        todo!("set_resource");
```

Comment:
is this meant to be fixed in this PR itself?

Reply:
Yes, it is removed now. We are relying on the default resources.

Reply:
Good catch, the change was leftover in my local. It is required, otherwise OTLP log exporter won't get the resource. I think we need unit-test for OTLP log exporter validate this. Let me see if I can add in this PR.